konfera/models/discount_code.py

```diff
@@ -23,6 +24,14 @@ class DiscountCode(FromToModel):
     def __str__(self):
         return self.title
 
+    def get_usage(self):
```

Comment:
How about we rename to something more obvious, like: `issued_tickets`
Also we can mar this function with `@property` decorator, this way it would be possible to call it `DiscountCode.issued_tickets`